include/lo2s/perf/event_reader.hpp

```diff
@@ -317,11 +318,11 @@ class EventReader
         return mmap_pages_ * get_page_size();
     }
 
-    char* data()
+    std::byte* data()
     {
         // workaround for old kernels
         // assert(header()->data_offset == get_page_size());
-        return (char*)base + get_page_size();
+        return reinterpret_cast<std::byte*>(base + get_page_size());
```

Comment:
```suggestion
        return reinterpret_cast<std::byte*>(base) + get_page_size();
```

The parenthesis were wrong.